### PR TITLE
merge with alex's repository

### PR DESCRIPTION
**Pull request by alexander-wise**
fixed a minor bug in function prepare_line_list_pass1
fixed a major bug in project_mask!
updated masks to be 2-column format with sorted wavelengths

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -7,6 +7,7 @@ version = "0.0.12"
 CSV = "336ed68f-0bac-5ca0-87d4-7b16caf5d00b"
 DSP = "717857b8-e6f2-59f4-9121-6e50c889abd2"
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
```
Comment (eford) [inline Project.toml:10]: What are you using from this package?

**Comment by eford** [comment]
Please remove the masks from the PR.  Those are data files for one specific project, rather than part of a general package.

File: Project.toml (hunk 1 above)
Comment (alexander-wise) [reply Project.toml:10]: SortedDict in the examples/alex_sandboxl (not included in push. I can remove that if you prefer)

**Comment by alexander-wise** [comment]
Do you know why these checks failed / what that means? They passed before, and all I did was remove the extra masks and change the Project.toml file which wasn't even using DataStructures...

File: Project.toml (hunk 1 above)
Comment (eford) [reply Project.toml]: Thanks.  You can add that to the Project.toml file for your example/sandbox code.

**Comment by eford** [comment]
> Do you know why these checks failed / what that means? They passed before, and all I did was remove the extra masks and change the Project.toml file which wasn't even using DataStructures...

Oh, I think I broke something between when you opened the PR and updated it.  I'll look into it.

**Review by eford — APPROVED** [review]

**Pull request merged**